1_fetch/src/get_nwis_data.R

```diff
@@ -32,14 +13,14 @@ download_nwis_site_data <- function(filepath, parameterCd = '00010', startDate="
     stringr::str_extract(pattern = "(?:[0-9]+)")
   
   # readNWISdata is from the dataRetrieval package
-  data_out <- readNWISdata(sites=site_num, service="iv", 
+  data_out <- readNWISdata(sites=site_num, 
+                           service="iv", 
                            parameterCd = parameterCd, startDate = startDate, endDate = endDate)
-
-  # -- simulating a failure-prone web-sevice here, do not edit --
+  #-- simulating a failure-prone web-sevice here, do not edit --
   if (sample(c(T,F,F,F), 1)){
     stop(site_num, ' has failed due to connection timeout. Try scmake() again')
   }
-  # -- end of do-not-edit block
+  #-- end of do-not-edit block
```

Comment:
This is an intriguing set of stylistic changes - what are your personal guiding principles/patterns for whitespace?

Reply:
I had uncommented/recommented this at one point, this was incidental